script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1C.txt

```diff
@@ -48,7 +48,7 @@ Although there was an unexpected visitor, my first morning here in seven years p
 <Page14>
 Hisui sees me off as I leave the grounds of the mansion.
 The Tohno residence is located on the edge of the city, on a hill away from the station.
-On the hill, there is a forest that is out of place in an urban area. Depending on how you look at it, the Tohno residence could be seen as a gate built at the entrance to that forest.
+On the hill, there is a forest that is rather out of place for an urban area. Depending on how you look at it, the Tohno residence could be seen as a gate built at the entrance to that forest.
```

Comment:
I believe out of place in is more common
![Uploading 2021-12-23-170127_1392x468_scrot.png…]()